conans/client/output.py

```diff
@@ -86,7 +87,10 @@ def write(self, data, front=None, back=None, newline=False):
         # Windows output locks produce IOErrors
         for _ in range(3):
             try:
-                self._stream.write(data)
+                if error:
```

Comment:
The opposite?

Reply:
yes, that explains some funny things I'm experiencing.